src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <React.Fragment>
+      <section className="Person">
+        <h2 className="Person__name">My name is {name}</h2>
+        {age === undefined ? null : <p className="Person__age">I am {age}</p>}
```

Comment:
You can use logical operator && here 
```suggestion
        {!!age && <p className="Person__age">I am {age}</p>}
```